modules/cnv-add-disk-to-vm.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:cnv-vm-add-disk:
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-disk-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-add-disk_{context}"]
+
+= Adding a virtual disk to a {object}
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Disks* tab.
+. Click *Add Disks* to open the *Add Disk* window:
```

Comment:
s/:/.

Reply:
Fixed. Thank you. Should be ready for merge and pick now to 4.2 and 4.3

Reply:
Hi @bmcelvee -  we have found a solution to get the merge to 4.2. If you can comment out the Ifdefs and Ifevals from modules/cnv-add-disk-to-vm.adoc it should fix it. Are you up for that? I wasn't aware that 4.1/2.1 and 4.2/2.2 are tethered in content, so the conditional code is causing builder to barf as the referenced stuff is not in 4.1/2.1

Copying @aburdenthehand

Reply:
@bmcelvee @bgaydosrh - just to be clear, only commenting out the ifevals that relate to 
ifeval::["{context}" == "cnv-edit-vms"]

The second one (cnv-editing-vm-template) can stay, and is needed to provide the {object} variables.

Reply:
Thanks! It's still not passing. I'll also tag Vikram to see if he might be able to provide some guidance.

Reply:
@bmcelvee - I noticed something else on that module and commented in your PR